src/main/resources/static/js/page/letsparty/detail.js

```diff
@@ -0,0 +1,186 @@
+let postNo = $("#postNo").val();
+
+// 최초 상세화면 이동시 댓글 2개 불러옴
+$(function() {
+	fetchAndRenderLatestTwoComments();
+});
+
+// 댓글 더보기 버튼 클릭시 전체 댓글 렌더링, 한 번 더 클릭하면 다시 최신 댓글 2개를 표시
+$(".more-button").click(function() {
+    let commentContainer = $("#comment-container");
+    if (commentContainer.hasClass("two-show")) { 
+		commentContainer.removeClass("two-show").addClass("all-show");
+	 	fetchAndRenderComments();
+		$(this).toggleClass("bi-chevron-down bi-chevron-up");
+	} else if(commentContainer.hasClass("all-show")) {
+		commentContainer.removeClass("all-show").addClass("two-show");
+		fetchAndRenderLatestTwoComments();
+		$(this).toggleClass("bi-chevron-down bi-chevron-up");
+	}
+});
+
+// 댓글쓰기 버튼을 누르면 댓글 등록 폼위치로 스크롤 이동
+$("#write-comment-btn").click(function(e) {
+	e.preventDefault();
+    $('html, body').animate({
+        scrollTop: $("#comment-form").offset().top
+    });
+});
+
+// 최근 댓글 2개만 불러오는 ajax 요청
+function fetchAndRenderLatestTwoComments() {
+    $.ajax({
+        url: "/letsparty/post/" + postNo + "/latest-two-comments",  // 최신 댓글 2개를 가져올 서버의 주소
+        method: "get"
+        })
+    .done(function(response) {
+        renderComments(response);
+    })
+    .fail(function(){
+		 alert("댓글을 가져오는 데 오류가 발생했습니다.");
+	});
+}
+		
+
+// 댓글 전체를 불러오는 ajax 요청
+function fetchAndRenderComments() {
+	$.ajax({
+	    url: "/letsparty/post/" + postNo + "/all-comments",  // 댓글 데이터를 가져올 서버의 주소
+	    method: "get"
+	})
+    .done(function(response) {
+        renderComments(response);
+    })
+    .fail(function(){
+		 alert("댓글을 가져오는 데 오류가 발생했습니다.");
+	});
+}
+
+// 게시물에 달린 댓글 수를 렌더링 하는 함수
+function renderCommentCnt(savedPost) {
+    $("#comment-count").text(savedPost.commentCnt);
+}
+
+// 댓글을 렌더링하는 함수
+function renderComments(comments) {
+    // 댓글이 성공적으로 등록된 경우 처리
+	// 날짜 변환 함수
+	function formatDate(dateString) {
+	    let date = new Date(dateString);
+	    let year = date.getFullYear();
+	    let month = date.getMonth() + 1;
+	    let day = date.getDate();
+	    let hours = date.getHours();
+	    let minutes = date.getMinutes();
+	    let seconds = date.getSeconds();
+	    
+	    // 시, 분, 초 한 자리 수 일 때 앞에 0을 붙임
+        hours = hours >= 10 ? hours : '0' + hours;
+        minutes = minutes >= 10 ? minutes : '0' + minutes;
+        seconds = seconds >= 10 ? seconds : '0' + seconds;
```

Comment:
```suggestion
        hours = String(hours).padStart(2, "0");
        minutes = String(minutes).padStart(2, "0");
        seconds = String(seconds).padStart(2, "0");
```
두 자리의 String이 되는 방법입니다.